src/components/prompts/FeedbackForm.js

```diff
@@ -69,13 +69,13 @@ class AskPermission extends React.Component {
         disabled = { this.state.submitType === null }
         color    ='teal'>
         {renderIfTrue(this.state.submitType === `withData`,
-          `Send With Data`
+          `Send it with my information`
         )}
         {renderIfTrue(this.state.submitType === `withoutData`,
-          `Send Without Data`
+          `Send it without my information`
         )}
         {renderIfTrue(this.state.submitType === null,
-          `Send`
+          `Send it`
```

Comment:
Not sure the "it" really adds anything - seems more awkward and I don't see how it would aid comprehension. I like "with/without my information" though.

Reply:
I just thought it would mesh better with the other two because they had 'Send it' at the start. But I'm not attached. I'll change it back.

Reply:
Oh, I meant for all three of them actually.

Reply:
Oh, hmm. I see what you're saying. Well, without 'it', it's not a full sentence. I'm not sure how much of a problem that would be for people. Still, I can take it out for now and if we can get our hands on some testing, we can just see how it goes.